app/assets/stylesheets/application.css.scss

```diff
@@ -15,6 +15,41 @@
  *= require font-awesome
  */
 
+#map-container { 
+  margin-top: 25px;
```

Comment:
Properties should be ordered border-color, border-radius, border-style, border-width, box-shadow, height, margin-top, margin-top, width